ontology/dprod/dprod-dcatprofile.ttl

```diff
@@ -140,281 +114,280 @@ dprod:DatasetShape
   .
 
-dprod:ProtocolShape
+dprod-shapes:ProtocolShape
     a sh:NodeShape;
-    rdfs:label "Protocol Shape" ;
-    dc:description "A protocol, possibly including a specific version, used for communicating with a service."@en ;
-    rdfs:isDefinedBy dprod:;
+    rdfs:label "protocol shape" ;
+    dct:description "A protocol, possibly including a specific version, used for communicating with a service."@en ;
+    rdfs:isDefinedBy dprod-shapes:;
     sh:targetClass dprod:Protocol;
 .
 
-dprod:SecuritySchemaTypeShape
+dprod-shapes:SecuritySchemaTypeShape
     a sh:NodeShape;
-    rdfs:label "Security Schema Type Shape" ;
-    dc:description "A security schema type used for authentication and communication."@en ;
-    rdfs:isDefinedBy dprod:;
+    rdfs:label "security schema type shape" ;
+    dct:description "A security schema type used for authentication and communication."@en ;
+    rdfs:isDefinedBy dprod-shapes:;
     sh:targetClass dprod:SecuritySchemaType;
 .
 
 #############################  dprod Property Shapes ################
 
-dprod:DataProduct-label
+dprod-shapes:DataProduct-label
   a sh:PropertyShape;
-  rdfs:isDefinedBy dprod:;
+  rdfs:isDefinedBy dprod-shapes:;
   sh:path rdfs:label;
   sh:datatype xsd:string;
-  dc:description "The name given to the data product."@en ;
-  rdfs:label "DataProduct-label";
+  dct:description "The name given to the data product."@en ;
+  rdfs:label "data product label shape";
 .
 
-dprod:DataProduct-description
+dprod-shapes:DataProduct-description
   a sh:PropertyShape;
-  rdfs:isDefinedBy dprod:;
-  sh:path dcterms:description;
+  rdfs:isDefinedBy dprod-shapes:;
+  sh:path dct:description;
   sh:datatype xsd:string;
-  dc:description "A free text description of the data product."@en ;
-  rdfs:label "DataProduct-description";
+  dct:description "A free text description of the data product."@en ;
+  rdfs:label "data product description shape";
 .
 
-dprod:DataProduct-type
+dprod-shapes:DataProduct-type
 a sh:PropertyShape;
-rdfs:isDefinedBy dprod:;
-sh:path dcterms:type;
-dc:description "The nature or genre of the data product."@en ;
-rdfs:label "DataProduct-description";
+rdfs:isDefinedBy dprod-shapes:;
+sh:path dct:type;
+dct:description "The nature or genre of the data product."@en ;
+rdfs:label "data product description shape";
 .
 
-dprod:DataProduct-dataProductOwner
+dprod-shapes:DataProduct-dataProductOwner
   a sh:PropertyShape;
-  rdfs:isDefinedBy dprod:;
+  rdfs:isDefinedBy dprod-shapes:;
   sh:path dprod:dataProductOwner;
-  sh:class foaf:Agent;
-  dc:description "The agent that is overall accountable for the data product. This includes managing the data product along its lifecycle ( creation, usage, versioning, deletion)."@en ;
-  rdfs:label "DataProduct-dataProductOwner";
+  sh:class prov:Agent;
+  dct:description "The agent that is overall accountable for the data product. This includes managing the data product along its lifecycle ( creation, usage, versioning, deletion)."@en ;
```

Comment:
Remove space in `( creation`

Reply:
deleted the instances here to avoid redundancy - the list in the ontology started Ideation